src/panels/lovelace/editor/config-elements/hui-entities-card-editor.ts

```diff
@@ -83,55 +91,60 @@ export class HuiEntitiesCardEditor extends LitElement
 
     return html`
       ${configElementStyle}
-      <div class="card-config">
-        <paper-input
-          .label="${this.hass.localize(
-            "ui.panel.lovelace.editor.card.generic.title"
-          )} (${this.hass.localize(
-            "ui.panel.lovelace.editor.card.config.optional"
-          )})"
-          .value=${this._title}
-          .configValue=${"title"}
-          @value-changed=${this._valueChanged}
-        ></paper-input>
-        <hui-theme-select-editor
-          .hass=${this.hass}
-          .value=${this._theme}
-          .configValue=${"theme"}
-          @value-changed=${this._valueChanged}
-        ></hui-theme-select-editor>
-        <div class="side-by-side">
-          <ha-formfield
-            .label=${this.hass.localize(
-              "ui.panel.lovelace.editor.card.entities.show_header_toggle"
-            )}
-            .dir=${computeRTLDirection(this.hass)}
-          >
-            <ha-switch
-              .checked=${this._config!.show_header_toggle !== false}
-              .configValue=${"show_header_toggle"}
-              @change=${this._valueChanged}
-            ></ha-switch>
-          </ha-formfield>
-          <ha-formfield
-            .label=${this.hass.localize(
-              "ui.panel.lovelace.editor.card.generic.state_color"
-            )}
-            .dir=${computeRTLDirection(this.hass)}
-          >
-            <ha-switch
-              .checked=${this._config!.state_color}
-              .configValue=${"state_color"}
-              @change=${this._valueChanged}
-            ></ha-switch>
-          </ha-formfield>
-        </div>
-      </div>
+      ${!this._hideGeneralEditor
```

Comment:
`hui-entities-card-editor` should either show the normal config or a detail config element, the detail should not be inside `hui-entities-card-row-editor`

Reply:
Wouldnt it makes sense if it was in the row editor. Since we are editing the row?

Reply:
Displaying multiple rows is something different than showing the editor of a row.

It would allow reusing the same for themes/actions etc.